client/layout/community-translator/launcher.jsx

```diff
@@ -86,7 +85,7 @@ class TranslatorLauncher extends React.PureComponent {
 					<h1>{ this.props.translate( 'Community Translator' ) }</h1>
 					<p>
 						{ this.props.translate(
-							'You have now enabled the translator.  Right click highlighted text to translate it.'
+							'You have now enabled the translator. Right click highlighted text to translate it.'
```

Comment:
:ok: This change is safe, we'll keep the existing 52 translations.

Reply:
I think we will want to update this text, since there is no highlighted text anymore.